build.cmd

```diff
@@ -518,8 +518,7 @@ if defined TF_BUILD (
     git fetch --all
 )
 
-set msbuildflags=%_nrswitch% /nologo
-REM set msbuildflags=%_nrswitch% /nologo
+set msbuildflags=%_nrswitch% /nologo /m
```

Comment:
Ah I should have checked the blame - I've been using it fine all day but we all know how flaky the CI is....